resources/python/pypi-info.py

```diff
@@ -1,24 +1,23 @@
-# This is a Python script that looks up package metadata on PyPI using
-# the XMLRPC API. It takes one argument, the name of the package (not
+# This is a Python script that looks up package metadata on PyPI.
+# It takes one argument, the name of the package (not
 # necessarily canonical), and outputs the results in JSON format (a
-# map, see pypiXMLRPCInfo). The script works on both Python 2 and
-# Python 3.
+# map). The script works on both Python 2 and Python 3.
 
 from __future__ import print_function
+import requests
```

Comment:
`requests` is not part of the standard library and so this script won't work on all systems. You can use `urllib.request` or IMO we shouldn't even be shelling out for this, let's do the request from within the upm process in Go

Reply:
@turbio is there a good reason you can think of why we're shelling out here?

Reply:
yeah, this seems like something we should just do in go land :shrug: 
I think it was original python since they've got a nice xmlrpc api

Reply:
But I like Python :(

Reply:
Fixed